### PR TITLE
Minor updates to cone search tutorial

### PR DESCRIPTION
**Pull request by pllim**
Add conesearch TODO. Update RADEC colnames.

This was done as part of investigation for #265, whose failure I cannot reproduce locally.

### CODE REVIEW

File: tutorials/notebooks/vo/conesearch.ipynb [hunk 1]
```diff
@@ -12,7 +12,13 @@
     "P. L. Lim\n",
     "\n",
     "## Learning Goals\n",
-    "* TODO\n",
+    "* Perform a cone search around M31 using an arbitrary service.\n",
```
Comment (adrn) [inline tutorials/notebooks/vo/conesearch.ipynb]: On first read, this phrasing is a little confusing - suggestion:
```suggestion
    "* Perform a cone search around M31 using a web service.\n",
```

File: tutorials/notebooks/vo/conesearch.ipynb [hunk 2]
```diff
@@ -12,7 +12,13 @@
...
+    "* Write the result out to a LaTeX table.\n",
+    "* Perform a SIMBAD query using cone search result.\n",
```
Comment (adrn) [inline tutorials/notebooks/vo/conesearch.ipynb]:
```suggestion
    "* Perform a SIMBAD query using the cone search result.\n",
```

File: tutorials/notebooks/vo/conesearch.ipynb [hunk 3]
```diff
@@ -12,7 +12,13 @@
...
+    "* Extract metadata from cone search catalog.\n",
```
Comment (adrn) [inline tutorials/notebooks/vo/conesearch.ipynb]:
```suggestion
    "* Extract metadata from the cone search catalog.\n",
```

File: tutorials/notebooks/vo/conesearch.ipynb [hunk 4]
```diff
@@ -12,7 +12,13 @@
...
+    "* Sort cone search result by angular distance.\n",
```
Comment (adrn) [inline tutorials/notebooks/vo/conesearch.ipynb]:
```suggestion
    "* Sort cone search results by angular distance.\n",
```

File: tutorials/notebooks/vo/conesearch.ipynb [hunk 5]
```diff
@@ -12,7 +12,13 @@
...
+    "* Search multiple cone search services at once (synchronously and asynchronously).\n",
+    "* Estimate cone search run time.\n",
```
Comment (adrn) [inline tutorials/notebooks/vo/conesearch.ipynb]:
```suggestion
    "* Estimate the run time of a cone search.\n",
```

**Comment by pllim** [comment]
Okie dokie. I accepted all your suggestions. Thanks for the review!

**Comment by adrn** [comment]
Can you rebase? The fix for the failing CI should be in master. Thanks!

**Comment by pllim** [comment]
Rebased and squashed.

**Comment by adrn** [comment]
Thanks @pllim!

**Pull request merged**